test/test.js

```diff
@@ -60,6 +66,12 @@ describe('Routes', () => {
       assert.strictEqual(typeof body.cid, 'string')
       assert.strictEqual(typeof body.providerAddress, 'string')
       assert.strictEqual(typeof body.protocol, 'string')
+
+      const { rows: [retrievalRow] } = await client.query(
+        'SELECT * FROM retrievals WHERE id = $1',
+        [body.id]
+      )
+      assert.strictEqual(retrievalRow.created_at_round, '42')
```

Comment:
I decided to add a test to verify that we are storing the correct round number in the database.